easybuild/easyconfigs/d/Doxygen/Doxygen-1.8.10-GNU-4.9.3-2.25.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'CMakeMake'
```

Comment:
ah, wait...

we have an easyblock for Doxygen, this seems to suggest we should update that with a version check, to use CMake for recent versions?